INFO     root:conftest.py:124 Starting test: test_basic_auth[wrong-wrong-Not authorized\n].
INFO     WDM:logger.py:11 xdist worker is: gw1
INFO     WDM:logger.py:11 ====== WebDriver manager ======
INFO     WDM:logger.py:11 Get LATEST chromedriver version for google-chrome
INFO     WDM:logger.py:11 Get LATEST chromedriver version for google-chrome
INFO     WDM:logger.py:11 Driver [/home/runner/.wdm/gw1/drivers/chromedriver/linux64/142.0.7444.59/chromedriver-linux64/chromedriver] found in cache
INFO     root:conftest.py:147 Started recording: tests_recordings/gw1/test_basic_auth[wrong-wrong-Not authorized\n]_20251029_072441/test_basic_auth[wrong-wrong-Not authorized\n]_20251029_072441.mp4
INFO     root:base_page.py:100 Navigating to URL: https://the-internet.herokuapp.com/.
INFO     root:base_page.py:102 Navigation completed.
INFO     root:base_page.py:82 Waiting for page to load with indicator '('css selector', 'h1.heading')' for 10s.
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
INFO     root:base_page.py:87 Page loaded successfully with indicator '('css selector', 'h1.heading')'.
INFO     root:page_manager.py:36 Navigating to Basic Auth page
INFO     root:basic_auth_page.py:17 Initialize URL based on username and password.
INFO     root:basic_auth_page.py:25 Navigate to URL: {url}.
INFO     root:base_page.py:100 Navigating to URL: http://wrong:wrong@the-internet.herokuapp.com/basic_auth.
INFO     root:base_page.py:102 Navigation completed.
INFO     root:basic_auth_page.py:30 Get authorization message.
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
ERROR    root:base_page.py:32 Timeout after 10s for locator ('css selector', 'div#content p'): Message: 
Stacktrace:
#0 0x55ec7e17baea <unknown>
#1 0x55ec7dbc7cdb <unknown>
#2 0x55ec7dc1a6c4 <unknown>
#3 0x55ec7dc1a901 <unknown>
#4 0x55ec7dc698b4 <unknown>
#5 0x55ec7dc66c87 <unknown>
#6 0x55ec7dc0caca <unknown>
#7 0x55ec7dc0d7d1 <unknown>
#8 0x55ec7e142ab9 <unknown>
#9 0x55ec7e145a8c <unknown>
#10 0x55ec7e12bd49 <unknown>
#11 0x55ec7e146685 <unknown>
#12 0x55ec7e1136c3 <unknown>
#13 0x55ec7e1687d8 <unknown>
#14 0x55ec7e1689b3 <unknown>
#15 0x55ec7e17aa83 <unknown>
#16 0x7f157849caa4 <unknown>
#17 0x7f1578529c6c <unknown>
INFO     root:conftest.py:151 Stopping video recording...
INFO     utils.video_recorder:video_recorder.py:95 Video created: tests_recordings/gw1/test_basic_auth[wrong-wrong-Not authorized\n]_20251029_072441/test_basic_auth[wrong-wrong-Not authorized\n]_20251029_072441.mp4
INFO     root:conftest.py:102 Quitting driver for browser: chrome.